doc/release-notes.md

```diff
@@ -51,249 +55,250 @@ masternodes. This results in PoSe punishment/banning for outdated masternodes,
+- `createwallet` now returns a warning if an empty string is used as an encryption password, and does not encrypt the wallet, instead of raising an error.
+  This makes it easier to disable encryption but also specify other options when using the `bitcoin-cli` tool.
+- The RPC `joinpsbts` will shuffle the order of the inputs and outputs of the resulting joined psbt.
+  Previously inputs and outputs were added in the order that the PSBTs were provided which makes correlating inputs to outputs extremely easy.
+- `importprivkey`: new label behavior.
+Previously, `importprivkey` automatically added the default empty label
+("") to all addresses associated with the imported private key.  Now it
+defaults to using any existing label for those addresses.  For example:
+
+  - Old behavior: you import a watch-only address with the label "cold
+    wallet".  Later, you import the corresponding private key using the
+    default settings.  The address's label is changed from "cold wallet"
+    to "".
+
+  - New behavior: you import a watch-only address with the label "cold
+    wallet".  Later, you import the corresponding private key using the
+    default settings.  The address's label remains "cold wallet".
+
+  In both the previous and current case, if you directly specify a label
+    during the import, that label will override whatever previous label the
+    addresses may have had.  Also in both cases, if none of the addresses
+    previously had a label, they will still receive the default empty label
+    ("").  Examples:
+  - You import a watch-only address with the label "temporary".  Later you
+    import the corresponding private key with the label "final".  The
+    address's label will be changed to "final".
+  - You use the default settings to import a private key for an address that
+    was not previously in the wallet.  Its addresses will receive the default
+    empty label ("").
+- The `createwallet`, `loadwallet`, and `unloadwallet` RPCs now accept
+  `load_on_startup` options to modify the settings list.
+- Several RPCs have been updated to include an "avoid_reuse" flag, used to control
+  whether already used addresses should be left out or included in the operation.
+  These include:
+  - `createwallet`
+  - `getbalance`
+  - `sendtoaddress`
+
+  In addition, `sendtoaddress` has been changed to enable `-avoidpartialspends` when
+  `avoid_reuse` is enabled.
+  The `listunspent` RPC has also been updated to now include a "reused" bool, for nodes
+  with "avoid_reuse" enabled.
+
+### Dash-specific changes in existing RPCs:
+- In rpc `upgradetohd` new parameter `rescan` was added which allows users to skip or force blockchain rescan. This params defaults to `false` when `mnemonic` parameter is empty and `true` otherwise.
 
 Please check `help <command>` for more detailed information on specific RPCs.
 
 Command-line options
 --------------------
+
 Most changes here were introduced through Bitcoin backports.
 
 New cmd-line options:
```

Comment:
If there are no new options, this line should either be removed or there should be a message on the next line indicating there are no new one.